### PR TITLE
Custom filename and directory

### PR DESCRIPTION
**Pull request by Grafikart**
For Android only at the moment (I plan to do this on iOS soon)
- Added a fileName option
- folderName can contain a full path

### CODE REVIEW

**Comment by VinceOPS** [comment]
+1. 

@JoschkaSchulz Could we consider this one? Thanks!

**Comment by JoschkaSchulz** [comment]
oh sorry :D haven't looked for a long time in this repo I will check and see if I'm allowed to merge it.

**Comment by VinceOPS** [comment]
Oh cool, thanks!

I quickly tried this plugin this morning. I like its simplicity, a lot. It worked on my Nexus 7 (Android 5+) but not on my Nexus 5 (Android 6+). I will have a loot at the debugs later, and will create an issue, if you still want to maintain the repo (do you?).

Thanks for sharing this :-) !

File: src/android/ImageResizer.java [hunk 1]
```diff
@@ -62,7 +66,7 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
         callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.OK, scaledFile.toString()));
         return true;
       } else {
-        callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.ERROR));
+          callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.ERROR));
```
Comment (salzig) [inline src/android/ImageResizer.java:69]: this indentation looks wrong.

**Comment by JoschkaSchulz** [comment]
I don't work at this company anymore and don't have the rights to merge anything... This pull requests looks good except the 4 spaces everywhere O.o 

Maybe @tiff can help?

File: src/android/ImageResizer.java (hunk 1 above)
Comment (JoschkaSchulz) [inline src/android/ImageResizer.java]: I don't work at this company anymore and don't have the rights to merge anything... This pull requests looks good except the 4 spaces everywhere O.o

Maybe @tiff can help?

**Comment by JoschkaSchulz** [comment]
I don't work at this company anymore and don't have the rights to merge anything... This pull requests looks good except the 4 spaces everywhere O.o

Maybe @tiff can help? Or if you don't want to maintain that, we can move that Repo on my account.

edit: exact that what @Salzig said.

**Comment by salzig** [comment]
maybe @colszowka can help.

**Comment by tiff** [comment]
@JoschkaSchulz you have access again :)

**Comment by JoschkaSchulz** [comment]
@tiff :+1: okay,... I correct fast the comment Salzig gave therefore the code looks clean.

**Pull request merged**